java/FoodTruckApp2/src/main/java/com/my/FoodTruckApp2/Customer/CustomerRepository.java

```diff
@@ -21,18 +21,26 @@
 public class CustomerRepository {
   @Autowired
   JdbcTemplate jdbcTemplate;
-  // ----------- INSERTING A NEW CUSTOMER -----------  //
-  public String createNewCustomer(@RequestBody CustomerRequestBody customerRequestBody){
-    String sql = "INSERT INTO customer(first_name,last_name) VALUES(?,?)";
+//   ----------- INSERTING A NEW CUSTOMER -----------  //
+  public Customer createNewCustomer(@RequestBody CustomerRequestBody customerRequestBody){
+    String sql = "INSERT INTO customer(first_name,last_name) VALUES(?, ?)";
     Integer rows = jdbcTemplate.update(sql,customerRequestBody.getFirstName(),customerRequestBody.getLastName());
+    String customer = "SELECT * FROM customer WHERE first_name = ? AND last_name = ?";
+    Customer newCustomer = jdbcTemplate.queryForObject(customer,
+        new BeanPropertyRowMapper<>(Customer.class),
+        customerRequestBody.getFirstName(),
+        customerRequestBody.getLastName());
+    // queryForObject lets the sql know the rows that is looking for will be put into an object
+    // the BeanPropertyRowMapper is shortcut because it will match the fields if its identical or close enough
     if(rows > 0){
-      log.info("A new customer was CREATED!!!(has been inserted)");
+      log.info("A new customer was CREATED!!!(has been inserted)" + newCustomer );
     }
-    return sql;
-    // the return goes straight to postman
-    // then goes to customerController line 41
+    return newCustomer;
+//     the return goes straight to postman
+//     then goes to customerController line 41
   }
 
```

Comment:
hey so these are not the changes we had agreed on yesterday

Reply:
Let me fix that

Reply:
Done. Is this the changes that  we agreed on yesterday ?

Reply:
yep